apps/condo/pages/contact/index.tsx

```diff
@@ -126,6 +128,10 @@ export const ContactsPageContent = ({
                                                                         accessCheck={canManageContacts}
                                                                         onFinish={refetch}
                                                                         columns={columns}
+                                                                        maxTableLength={hasFeature('bigger_limit_for_contacts_import') ?
```

Comment:
How about common feature-flag for meters and contacts? Seems like it be more comfortable....and people who load meters, load contacts as well

Reply:
added common feature flag, updated .helm